pyth-sdk/src/price.rs

```diff
@@ -346,10 +605,43 @@ impl Price {
             (x as u64, 1)
         }
     }
+
+    /// Helper function to create fraction
+    /// 
+    /// fraction(x, y) gives you the normalized Price closest to x/y.
+    /// If you cannot represent x/y exactly within 8 digits of precision, it may zero out the remainder.
+    /// In particular, if x and/or y cannot be represented within 8 digits of precision, potential for precision error.
+    /// If x and y can both be represented within 8 digits of precision AND x/y can be represented within 8 digits, no precision loss.
```

Comment:
this comment doesn't match the implementation. The implementation actually gives you the unnormalized price that's the best approximation of x/y. (division doesn't return a normalized number)

this implementation is fine, but please fix the comment. Also suggest adding that the result can be in an arbitrary exponent and callers may need to scale it using scale_to_exponent.